cogs/countdown.py

```diff
@@ -1,31 +1,54 @@
 import logging
 import datetime
 import asyncio
+import random
 from discord.ext import tasks, commands
-from constants import ANNOUNCEMENTS_CHAN_ID
+from constants import ANNOUNCEMENTS_CHAN_ID, HYPE_MSGS
 
 logger = logging.getLogger(__name__)
 
 
 class Countdown(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
-        year = datetime.datetime.now().year
-        self.bitcoin_pizza_day = datetime.date(year, 5, 22)  # May 22, 2021 etc
+        year = datetime.datetime.utcnow().year
+        self.bitcoin_pizza_day = datetime.date(year, 5, 22)
+        self.tau_day = datetime.date(year, 6, 28)
         self.target_hour = 15  # 3pm UTC / 11am EDT / 10am CDT / 8am PDT
-        self.message = "Hello @everyone!! Today is {date}, which means there are **{days} DAYS until May 22**, Bitcoin Pizza Day! Let's goooo!"
+        self.messages = {
```

Comment:
These messages and the dates they are attached to don't make sense. 

1. The messages themselves could be a function. Especially since it's just the date and the name of the event. 
2. The dates should all be constants set somewhere else since they are never being reset. There could just be a file of all the date constants

```
datetime.datetime.utcnow().year
BITCOIN_PIZZA_DAY = datetime.date(year, 5, 22)
TAU_DAY = datetime.date(year,6, 28)
```